x/mint/keeper/export_test.go

```diff
@@ -0,0 +1,11 @@
+package keeper
+
+const (
+	DeveloperVestingAmount = developerVestingAmount
+)
+
+var (
+	ErrAmountCannotBeNilOrZero               = errAmountCannotBeNilOrZero
+	ErrDevVestingModuleAccountAlreadyCreated = errDevVestingModuleAccountAlreadyCreated
+	ErrDevVestingModuleAccountNotCreated     = errDevVestingModuleAccountNotCreated
+)
```

Comment:
nit: 
IMO, I don't think errors have to private variables that need secondary export for tests. 
what do you think about moving these to `errors.go` as other packages do and making them public functions?

Reply:
These exports are only compiled for tests. Any file that ends with `_test.go` does not get compiled into the regular binary.

I think a good practice is to have package-private variables and constants if they are not used anywhere else. If we see that we reuse the same variables/errors across multiple packages, then, I agree, it constitutes the need for moving into global scope / made exported. Let me know what you think, happy to discuss further